DockerizeJenkinsfile

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/env groovy
 
+def use_release_artifacts() {
+    return (env.BRANCH_NAME ==~ /^release_(\d+)\.(\d+)\.(\d+)$/) ? 'true' : 'false'
+}
+
 pipeline {
     agent {
         docker {
-            image 'registry.gear.ge.com/dig-predix-security-services/uaa-ci-testing-with-docker:latest'
+            image 'registry.gear.ge.com/dig-predix-security-services/uaa-ci-testing:latest'
             label 'dind'
             args '-v /var/lib/docker/.gradle:/root/.gradle -v /var/run/docker.sock:/var/run/docker.sock --add-host "localhost":127.0.0.1'
         }
     }
     environment {
+        USE_RELEASE_ARTIFACTS = true
```

Comment:
Release artifacts should be used only when on release branch.  Use use_release_artifacts() to assign.
It was temporary forced to true on release_4.30.0_dockerized branch.

Reply:
Fixed in last commit